### PR TITLE
net-nntp/slrn: taking ownership + version bump

### PR DESCRIPTION
**Pull request by nemunaire**
* Using EAPI=6
* Fix repoman complaint about openssl missing slot

```diff
--- /usr/portage/net-nntp/slrn/slrn-1.0.2.ebuild	2017-02-28 20:50:50.000000000 +0100
+++ /usr/portage/net-nntp/slrn/slrn-1.0.3.ebuild	2017-08-13 18:51:08.031835919 +0200
@@ -1,41 +1,36 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
-inherit autotools-utils
+EAPI=6
 
-MY_P="${PN}_${PV/_/~}"
+MY_P="${P}a"
 
 DESCRIPTION="A s-lang based newsreader"
 HOMEPAGE="http://slrn.sourceforge.net/"
-SRC_URI="http://jedsoft.org/releases/${PN}/${P}.tar.bz2"
+SRC_URI="http://jedsoft.org/releases/${PN}/${MY_P}.tar.bz2"
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="amd64 ppc x86 ~amd64-linux ~x86-linux"
+KEYWORDS="~amd64 ~ppc ~x86 ~amd64-linux ~x86-linux"
 IUSE="canlock nls ssl uudeview"
 
 RDEPEND="virtual/mta
 	app-arch/sharutils
-	>=sys-libs/slang-2.1.3
+	>=sys-libs/slang-2.2.3
 	canlock? ( net-libs/canlock )
-	ssl? ( dev-libs/openssl )
+	ssl? ( dev-libs/openssl:0= )
 	uudeview? ( dev-libs/uulib )"
 DEPEND="${RDEPEND}
 	nls? ( sys-devel/gettext )"
 
-AUTOTOOLS_IN_SOURCE_BUILD=1
 PATCHES=( "${FILESDIR}"/${P}-make.patch )
 
 src_configure() {
-	local myeconfargs=(
-		--with-docdir="${EPREFIX}"/usr/share/doc/${PF}
-		--with-slrnpull
-		$(use_with canlock)
-		$(use_with uudeview uu)
-		$(use_enable nls)
+	econf \
+		--with-docdir="${EPREFIX}"/usr/share/doc/${PF} \
+		--with-slrnpull \
+		$(use_with canlock) \
+		$(use_with uudeview uu) \
+		$(use_enable nls) \
 		$(use_with ssl)
-	)
-
-	autotools-utils_src_configure
 }
```

### CODE REVIEW

File: net-nntp/slrn/slrn-1.0.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86 ~amd64-linux ~x86-linux"
+IUSE="canlock nls ssl uudeview"
+
+RDEPEND="virtual/mta
+	app-arch/sharutils
+	>=sys-libs/slang-2.2.3
+	canlock? ( net-libs/canlock )
+	ssl? ( dev-libs/openssl:0= )
```
Comment (lorem-ipsum) [inline net-nntp/slrn/slrn-1.0.3.ebuild]: You don't need the 0 slot specifier, just `:=` should be fine.
Would you mind to add libressl support as well? You can find a description of the USE syntax for it [here](https://wiki.gentoo.org/wiki/Project:LibreSSL).

File: net-nntp/slrn/slrn-1.0.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_P="${P}a"
```
Comment (mgorny) [inline net-nntp/slrn/slrn-1.0.3.ebuild]: Why not name the ebuild `-1.0.3a`? It's a valid Gentoo version.

File: net-nntp/slrn/slrn-1.0.3.ebuild [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+DESCRIPTION="A s-lang based newsreader"
+HOMEPAGE="http://slrn.sourceforge.net/"
+SRC_URI="http://jedsoft.org/releases/${PN}/${MY_P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86 ~amd64-linux ~x86-linux"
+IUSE="canlock nls ssl uudeview"
+
+RDEPEND="virtual/mta
```
Comment (mgorny) [inline net-nntp/slrn/slrn-1.0.3.ebuild]: Please sort the unconditional dependencies lexically, it's easier to review when we can expect the order.

File: net-nntp/slrn/slrn-1.0.3.ebuild (hunk 2 above)
Comment (nemunaire) [inline net-nntp/slrn/slrn-1.0.3.ebuild]: This `a` is only present in the tarball name. Once extracted, the directory is called `slrn-1.0.3`, without `a` and all communications are made without the trailing `a`. I don't know why there is this `a` in the tarball name...

Debian came back from using that `a` as a version identifier: https://sources.debian.net/src/slrn/1.0.3%2Bdfsg-1/debian/changelog/

File: net-nntp/slrn/slrn-1.0.3.ebuild (hunk 3 above)
Comment (nemunaire) [inline net-nntp/slrn/slrn-1.0.3.ebuild]: Done :-)

File: net-nntp/slrn/slrn-1.0.3.ebuild (hunk 2 above)
Comment (mgorny) [inline net-nntp/slrn/slrn-1.0.3.ebuild]: Okay then.

**Comment by candrews** [comment]
Thanks!

dba981db737f5c120a553113aff6bc34ea2c33ae
1e4e32e6ad7047ec180ffcb6b92fe79d8338259c

**Pull request closed without merging**